src/statsmodel.jl

```diff
@@ -106,12 +106,50 @@ function formula end
 formula(m::TableStatisticalModel) = m.mf.f
 formula(m::TableRegressionModel) = m.mf.f
 
+"""
+    termnames(model::StatisticalModel)
+    termnames(term::AbstractTerm)
+
+Return the names associated with terms associated with a model.
```

Comment:
```suggestion
Return the names of terms used in the formula of `model`.
```
It would be useful to specify whether this always returns a tuple, or whether for models with no response (e.g. PCA) this will return a vector without wrapping it in a tuple.